apps/docs/security/delete-protection.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Delete Protection
+description: Prevents an API from being deleted when enabled.
+---
+
+# Delete Protection
+
+Delete Protection is a safety feature that prevents accidental deletion of your APIs. When enabled, it blocks all deletion attempts through both the dashboard and API.
+
+## Overview
+
+Delete Protection adds an extra layer of security to your APIs by:
+
+- Preventing accidental deletion of critical APIs
+- Requiring explicit confirmation to disable protection
+- Creating an audit trail of protection changes
+- Blocking deletion attempts through both UI and API
+
+## Enabling Delete Protection
+
+1. Navigate to your API settings in the dashboard
+2. Click "Enable Delete Protection" to activate protection
+3. Type the API name to confirm
+4. Click "Enable API Delete Protection" to add protection 
+
+Once enabled, the API cannot be deleted until protection is explicitly disabled.
+
+## Disabling Delete Protection
+
+To disable Delete Protection:
+
+1. Navigate to your API settings in the dashboard
+2. Click "Disable Delete Protection"
+3. Type the API name to confirm
+4. Click "Disable API Delete Protection" to remove protection
+
+## API Behavior
+
+When Delete Protection is enabled:
+
+- All deletion attempts are blocked
+- The API returns a `DELETE_PROTECTED` error
+- The error includes a link to this documentation
+
+### Example Error Response
+
+```json
+{
+  "error": {
+    "code": "DELETE_PROTECTED",
+    "docs": "https://unkey.com/docs/api-reference/errors/code/DELETE_PROTECTED",
+    "message": "api [apiId] is protected from deletions",
+    "requestId": "req_1234"
+  }
+}
+```
+
+## Audit Trail
+
+All changes to Delete Protection are logged in the audit trail, including:
```

Comment:
they're not actually. no audit logs are produced here.
we should do this though in the future